infra_tooling_account/infra_tooling_account/grafana/glue_jobs_dashboard.template.json

```diff
@@ -19,6 +19,66 @@
   "links": [],
   "liveNow": false,
   "panels": [
+    {
+      "datasource": "Amazon-Timestream",
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "thresholds"
+          },
+          "mappings": [],
+          "noValue": "no data",
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              }
+            ]
+          }
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 14,
+        "w": 3,
+        "x": 0,
+        "y": 0
+      },
+      "id": 2,
+      "options": {
+        "colorMode": "value",
+        "graphMode": "area",
+        "justifyMode": "auto",
+        "orientation": "horizontal",
+        "reduceOptions": {
+          "calcs": [
+            "lastNotNull"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "text": {
+          "titleSize": 20,
+          "valueSize": 40
+        },
+        "textMode": "value_and_name",
+        "wideLayout": false
+      },
+      "pluginVersion": "10.2.2",
+      "targets": [
+        {
+          "database":  "<<DATABASE_NAME>>",
+          "datasource": "Amazon-Timestream",
+          "measure": "execution",
+          "rawQuery": "SELECT   sum(execution) as \"Total runs\", sum(failed) as \"Failed\",sum(succeeded) as \"Succeeded\" , round(sum(dpu_seconds/60), 2) as \"DPU, min\" ,round(sum(execution_time_sec/60), 2) as \"Execution time,min\" FROM $__database.$__table where $__timeFilter and resource_name in (${resource_name:singlequote}) and monitored_environment in (${monitored_env:singlequote})\r\n",
```

Comment:
Let's measure DPU consumption and Execution time in hours. I guess, that's more suitable for larger scale in actual envs (as opposed to our tiny testing stand).

and also, I'd call metric "DPU-hours" as it aligns with glue pricing aspect name